Triton_Inference_Server_Python_API/examples/rayserve/README.md

```diff
@@ -0,0 +1,18 @@
+# Triton Inference Server Ray Serve Deployment
+
+Using the Triton Inference Server In-Process Python API you can easily
+integrate your triton server based models into any Python framework
+including FastAPI and Ray Serve.
+
+This directory contains a simple Triton Inference Server based Ray Serve
+deployment.
+
```

Comment:
Can you add more descriptions regarding the ray serve deployment? What are the endpoints? What is the model being served? S3 bucket requirements etc.

Reply:
added additional information